spec/rails_helper.rb

```diff
@@ -8,13 +8,28 @@
 require 'capybara/rails'
 require 'capybara/rspec'
 require 'capybara-screenshot/rspec'
+require 'selenium/webdriver'
 
 ActiveRecord::Migration.maintain_test_schema!
 
 Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
 
+Capybara.register_driver :chrome do |app|
+  Capybara::Selenium::Driver.new(app, browser: :chrome)
+end
+
+Capybara.register_driver :headless_chrome do |app|
+  capabilities = Selenium::WebDriver::Remote::Capabilities.chrome(
+    chromeOptions: { args: %w(headless disable-gpu) }
+  )
+
+  Capybara::Selenium::Driver.new app,
+                                 browser: :chrome,
+                                 desired_capabilities: capabilities
+end
+
 Capybara.default_driver = :webkit
-Capybara.javascript_driver = :webkit
+Capybara.javascript_driver = :headless_chrome
```

Comment:
For some reason this didn't actually launch a headless instance on my local machine. Not sure why, not a huge deal, but thought I'd mention it.